demo/src/app/components/modal/modal.component.ts

```diff
@@ -0,0 +1,12 @@
+import {Component} from '@angular/core';
+import {NgbdApiDocs} from '../shared/api-docs';
+
+@Component({
+  selector: 'ngbd-modal',
+  template: `
+    <ngbd-api-docs directive="NgbModalWindow"></ngbd-api-docs>
+    <ngbd-api-docs directive="NgbModalBackdrop"></ngbd-api-docs>
```

Comment:
We don't want to document `NgbModalWindow`, `NgbModalBackdrop` as those are internal components not exposed to users. Remove.